lib/session/session.go

```diff
@@ -110,6 +110,48 @@ type Session struct {
 	Moderated bool `json:"moderated"`
 }
 
+// FileTransferParams contain parameters for requesting a file transfer
+type FileTransferParams struct {
+	RequestID string `json:"requestId"`
+	Approved  bool   `json:"approved"`
+	// Response denotes if this is a response to a file transfer request.
+	// If true, the Approved field will be set. True = approved
+	Response bool `json:"response"`
+	// Direction is either upload or download
+	Direction string `json:"direction"`
+	// Location is location of file to download, or where to put an upload
+	Location string `json:"location"`
+	// Filename is the name of the file to be uploaded
+	Filename string `json:"filename"`
+	// Size is the size of the file to be uploaded
+	Size string `json:"size"`
+	// Requster is the authenticated Teleport user who requested the file transfer
+	Requester string `json:"requester"`
+	// Approvers is a list of teleport users who have approved the file transfer request
+	Approvers []Party `json:"approvers"`
```

Comment:
Why do you need an `Approved` field if you have approvers? Is it possible for `Approved` to be false but `Approvers` to be non-empty?

Reply:
This is a bad consequence of having the request/response in this single struct. What `Approved` actually meant was "is this response and approval or deny", which now that I've seen through the weeds is NOT very clear. Removing it from this and making a discrete `Response` struct will clear it up